_episodes/licenses.md

```diff
@@ -1,24 +1,149 @@
+>- **FIXME**: check what the answer should be
+{: .discussion}
+
+Further reading:
+- https://gmd.copernicus.org/articles/11/3659/2018/ (Section 4)
+
+
+
+Data ownership
+--------------
+
+Possible owners of the data:
+- You
+- Institute you work for
+- Funding agency
+
+
+You may be restricted by your institutions’ policies on licenses
+
+Data ownership is defined in the data management plan
+
+
+> ## Excercise
+>Take a few minutes to find out whether your institute has a license policy.
+>
+>- What does it say?
+>- Who owns *your* data?
+{: .discussion}
+
+> ## Excercise
+>- Imagine you have downloaded some data from source X and applied operation Y on it.
+>- Which data licence should you put on it when you distribute the data?
+>- Which licence can you not put on it when you distribute the data?
+{: .discussion}
+
+
+What does the EU require?
+-------------------------
+
+The European Commission has fully embraced the FAIR principles. Horizon2020 already mandates open access to all scientific publications, and is now running a pilot from 2017 for research data to be open by default with possibilities to opt out.
+
+How the data will be licenced is part of the data management plan. Specificically:
+
+- How will the data be licensed to permit the widest re-use possible?
+- When will the data be made available for re-use (i.e. are the data under embargo)?
+- Are the data usable by third parties (i.e. are there restrictions)?
+
+In all cases, the Commission encourages researchers to provide access to their data in the broadest sense. They encourage authors to retain their copyright, but grant adequate licences to publishers. They recommend the Creative Common licences as a useful legal tool for providing open access.
+
+Further reading:
+- <https://ec.europa.eu/research/participants/docs/h2020-funding-guide/cross-cutting-issues/open-access-data-management/open-access_en.htm>
+- <https://ec.europa.eu/research/participants/data/ref/h2020/other/hi/oa-pilot/h2020-infograph-open-research-data_en.pdf>
+- In case you are really interested, have a look at [p251-253 of the Horizon2020 Programme](https://ec.europa.eu/research/participants/data/ref/h2020/grants_manual/amga/h2020-amga_en.pdf)
+
+How are licences applied to research data?
+------------------------------------------
 
-CMIP5 and CMIP6 licenses (special case, example)
+Adding a licence to data is easy. You just need to mention or refer to the licence in the same place where you upload your data. Some repositories like Zenodo allow you to select a licence, and they make sure it is clear under which licence the data are available.
 
-License restrictions (share alike etc.) for derived data
+You can apply a licence by one of these ways:
 
-Institutions’ policies on licenses
+- Choosing a licence when you upload your data to a repository
+- Referring to the licence on the webpage where the data are hosted
+- Attaching the licence to the metadata
+- Adding a readme file / licence file to your data
```

Comment:
Note: some projects (e.g. CMIP6) explicitly require you to add the licence to the metadata.